package.json

```diff
@@ -13,12 +13,12 @@
     "build:polygon": "graph build --network matic",
     "build:sepolia": "graph build --network sepolia",
     "deploy:goerli": "graph deploy --product hosted-service otterspace-xyz/badges-goerli --network goerli",
-    "deploy:goerli-alpha": "graph deploy --product hosted-service otterspace-xyz/badges-goerli-alhpa --network goerli",
+    "deploy:goerli-alpha": "graph deploy --product hosted-service otterspace-xyz/badges-goerli-alpha --network goerli",
     "deploy:optimism": "graph deploy --product hosted-service otterspace-xyz/badges-optimism --network optimism",
     "deploy:optimism-alpha": "graph deploy --product hosted-service otterspace-xyz/badges-optimism-alpha --network optimism",
     "deploy:optimism-goerli": "graph deploy --product hosted-service otterspace-xyz/badges-optimism-goerli --network optimism-goerli",
     "deploy:polygon": "graph deploy --product hosted-service otterspace-xyz/badges-polygon --network matic",
-    "deploy:sepolia": "graph deploy --product=subgraph-studio badges-sepolia --network=sepolia --version-label=1.0.0",
+    "deploy:sepolia": "graph deploy --product=subgraph-studio badges-sepolia --network=sepolia --version-label=1.0.1",
```

Comment:
does changing the version also change the url of SG? of so you might have to update the API staging config for it

Reply:
the url stays the same here. But they introduced version flag being compulsory for deployments to their Studio environment